### PR TITLE
feat: HOME-49 admin panel

### PR DESCRIPTION
**Pull request by mateusz-kleszcz**

### CODE REVIEW

File: .storybook/stories/Typography/Typography.tsx
Comment (Phabes) [inline .storybook/stories/Typography/Typography.tsx:1]: `selectable` will be usefull

File: views/AdminView/AdminView.styles.ts
Comment (Phabes) [inline views/AdminView/AdminView.styles.ts:1]: `display: flex` is default in React Native so we can skip it.

File: views/AdminView/components/BotNameInput.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        <Typography
+          variant="body-medium"
+          text="Nazwa jest juz zajęta. Spróbouj ponownie"
+          color="text-error"
+        />
+      )}
+      {isUsernameInvalidError && (
+        <Typography
+          variant="body-medium"
+          text="Nazwa jest nieprawidłowa. Spróbouj ponownie"
```
Comment (Phabes) [inline views/AdminView/components/BotNameInput.tsx:60]:
```suggestion
          text="Nazwa jest nieprawidłowa. Spróbuj ponownie"
```

File: views/AdminView/components/BotNameInput.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        placeholder="Podaj nazwę lokalizacji..."
+        variant={
+          isUsernameInvalidError || isUsernameTakenError ? 'error' : 'default'
+        }
+        disabled={!isWaitingForUsername || isSubmitting}
+      />
+      {isUsernameTakenError && (
+        <Typography
+          variant="body-medium"
+          text="Nazwa jest juz zajęta. Spróbouj ponownie"
```
Comment (Phabes) [inline views/AdminView/components/BotNameInput.tsx:53]:
```suggestion
          text="Nazwa jest juz zajęta. Spróbuj ponownie"
```

File: views/AdminView/components/BotNameInput.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    setIsSubmitting(true);
+    await sendIoTMessage(botUsername, botFatherAccessHash, botFatherId);
+    setIsSubmitting(false);
+  }, [botUsername, botFatherAccessHash, botFatherId]);
+
+  return (
+    <>
+      <Typography
+        variant="body-medium"
+        text="Podaj username dla bota, którego będziesz uzywał na Raspberry PI. Jego nazwa musi kończyć się na bot, np. HomeBot lub home_bot"
```
Comment (Phabes) [inline views/AdminView/components/BotNameInput.tsx:38]: I would add e.g. `_bot` at the end without requesting it from user

File: views/LocationListView/LocationListView.styles.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { StyleSheet } from 'react-native';
+import { theme } from '../../.storybook/theme';
+
+export const styles = StyleSheet.create({
+  container: {
+    display: 'flex',
```
Comment (Phabes) [inline views/LocationListView/LocationListView.styles.ts:6]: Same here. Delete `display: flex`.

**Review by Phabes — APPROVED** [review]
GJ

File: .storybook/stories/Typography/Typography.tsx
Comment (mateusz-kleszcz) [reply .storybook/stories/Typography/Typography.tsx:1]: It's added

File: views/AdminView/components/BotNameInput.tsx (hunk 3 above)
Comment (mateusz-kleszcz) [reply views/AdminView/components/BotNameInput.tsx:38]: We can do this in the future, but for now, the user has to provide the bot name in the `.env` file, so I don't want to make it inconsistent.

**Pull request merged**